src/Entity/Transformer.php

```diff
@@ -179,7 +179,7 @@ public function loadBlock(array $content, PageInterface $page): PageBlockInterfa
             $block->setName($content['name']);
         }
 
-        $block->setEnabled($content['enabled']);
+        $block->setEnabled(\in_array($content['enabled'], ['1', true], true));
```

Comment:
Next major can be added here too no ?

Reply:
Does it matter tho? phpstan and psalm will comply when you change the type, also in test could be added too, but its the same. Do you prefer adding NEXT_MAJOR on all places?

Reply:
They won't complain here: https://phpstan.org/r/a02bf6cd-33d1-407a-a39c-b0a06b9e8023

Reply:
Alright, added!.